src/css/components/header-buttons-mobile.css

```diff
@@ -1,22 +1,34 @@
 .js-overlay-active .mobile-buttons {
   position: fixed;
   top: 1rem;
-  right: 1rem;
+  right: 0;
   z-index: 10;
   left: 0;
-  right: 0;
   background: white;
   height: var(--sp3);
   top: 0;
   padding-right: 1rem;
+  @media (--md) {
+    top: 0;
+    height: 108px;
+    left: initial;
+    width: 100%;
+    max-width: 500px;
```

Comment:
Add a comment to what this 500px is. Right now it's a magic number.

Reply:
Haha, no is not a magic number, it comes from `header.css` on line `100`

Reply:
Yeah I know, but we need to ensure that other people reading this code realize that too.